Cython/Compiler/Code.pxd

```diff
@@ -25,6 +26,7 @@ cdef class UtilityCode(UtilityCodeBase):
     cdef public list specialize_list
     cdef public object file
     cdef public tuple _parts_tuple
+    cdef public list shared_utility_functions
```

Comment:
I don't think this needs to be `public`. (But then, `_parts_tuple` probably doesn't either – nor a few others, I guess. So maybe not something to clean up in this PR.)